std_workflow/js/std_workflow_emails.js

```diff
@@ -335,7 +335,7 @@ var templateFunctionRenderList = function(t, items) {
 };
 
 
-P.globalTemplateFunction("M:entity-title", function(object) {
+P.globalTemplateFunction("M:toUser-title", function(object) {
```

Comment:
Naming is hard...
Best I can come up with it M:person-name-title
as I sort of see where you're going with toUser, but it's more generic than that, and now gives you camel case in a function name which is inconsistent with everything else.
I'm trying to match 'first-name' and 'last-name' in style.

Reply:
Function name changed.